### DISCUSSION TITLE
OKD on VM and installation on the same HW

### DISCUSSION THREAD

**Discussion by BCollPerales** [okd-project/okd#1248] [General]
Dear colleagues, I'm new to OKD and I have some doubts (I fear they might be basic one). I have reviewed the documentation and I'm still not sure about the OKD cluster installation. According to the attached file, OKD is made of a Master Node, Node(s) and Infra Node(s). Could you please tell me whether the installation can be made 1) on a single bare-metal server (both the master and the nodes); 2) on different bare-metal servers (e.g. one for the master and one for the nodes)??

![ArchOKD](https://user-images.githubusercontent.com/83281466/171352852-8dc3c6b6-b3fd-4e58-9abb-d0bb78e74311.png)

Thanks in advance for your support!

**Comment by vrutkovs** [comment]
>1) on a single bare-metal server (both the master and the nodes)

Yes. There are multiple options here - run several VMs in libvirt or a single master/worker node via Assisted Installer

>2) on different bare-metal servers (e.g. one for the master and one for the nodes)??

Yes
